src/Reflection/ReflectionProperty.php

```diff
@@ -265,6 +304,16 @@ public function isStatic(): bool
         return ($this->modifiers & CoreReflectionProperty::IS_STATIC) === CoreReflectionProperty::IS_STATIC;
     }
 
+    public function isFinal(): bool
+    {
+        return ($this->modifiers & ReflectionPropertyAdapter::IS_FINAL_COMPATIBILITY) === ReflectionPropertyAdapter::IS_FINAL_COMPATIBILITY;
+    }
+
+    public function isAbstract(): bool
+    {
+        return ($this->modifiers & ReflectionPropertyAdapter::IS_ABSTRACT_COMPATIBILITY) === ReflectionPropertyAdapter::IS_ABSTRACT_COMPATIBILITY;
```

Comment:
You should be able to cast to `(bool)` without using `===`. Should apply to the rest of equalities in this API :thinking:

Reply:
Yes, PHPStan needs it currently.

Reply:
Simplified and added other commit to simply other equalities.